test/settings.js

```diff
@@ -50,23 +50,6 @@ module.exports.tests.analysis = function(test, common) {
 
 // -- analyzers --
 
-// this multiplexer filter provides all the synonyms used by the peliasAdmin analyzer
```

Comment:
these tests just got moved down the file.
they were listed under `analyzer` when they're really a `token_filter`